<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgPreferencesPageHome</class>
 <widget class="QWidget" name="WdgPreferencesPageHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="gbUpdate_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lTitleIconLanguage">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/gear.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="cbLanguage"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbUpdate_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lTitlteIconTheme">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/sun-moon.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="cbStyle"/>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbAppearance">
          <property name="text">
           <string>Go to appearance</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbUpdate_4">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lTitleIconKeyboardShortcuts">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/keyboard.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lTitleTextKeyboardShortcuts">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Keyboard Shortcuts</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbKeyboardShortcuts">
          <property name="text">
           <string>Go to shortcut editor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbUpdate_7">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="lTitleIconTroubleshootingDebugging">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/bug.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lTitleTextTroubleshootingDebugging">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Troubleshooting &amp; Debugging</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbOpenLogfileLocation">
          <property name="text">
           <string>Open Logfile Location</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbConfigEditor">
          <property name="text">
           <string>Open Config Editor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbResetAllPreferences">
          <property name="text">
           <string>Reset All Preferences</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="gbUpdate">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lTitleIconUpdates">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/Updates.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lTitleTextUpdates">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Updates</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lUpdatesInfo">
          <property name="text">
           <string>ScheduleMaster is up to date!</string>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbUpdateSettings">
          <property name="text">
           <string>Update settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbUpdate_6">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lTitleIconPlugins">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/puzzle-piece.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lTitleTextPlugins">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Plugins</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lPluginsDescription">
          <property name="text">
           <string>Extend ScheduleMasters features using plugins</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbPluginManager">
          <property name="text">
           <string>Go to plugin manager</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbUpdate_5">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lTitleIconAbout">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../Resources.qrc">:/Icons/circle-info.ico</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lTitleTextAbout">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>About ScheduleMaster</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="llVersion">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lVersion">
            <property name="text">
             <string>&lt;Version&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
